pkg/fileHandling.go

```diff
@@ -0,0 +1,82 @@
+package pkg
```

Comment:
do not use camel case in the file name, Instead of fileHandling.go you can use  file.go only